static/beta/itless/services/services.json

```diff
@@ -25,8 +24,7 @@
         "title": "Red Hat Enterprise Linux",
         "links": [
           "rhel.inventory",
-          "rhel.groups",
-          "rhel.workspaces"
+          "rhel.groups"
```

Comment:
Same question here

Reply:
Because its in the "title": "Identity and Access Management",